### ISSUE TITLE
Cannot login after upgrade Ghost

### ISSUE THREAD

**Issue by baddwin** [vickychijwani/quill#146]
FILL THESE DETAILS FIRST:

- Ghost version: 0.7.1
- App version: 1.3.3
- Device name: Asus Zenfone ZE550KL

----

I've just updated my Ghost server from 0.6.x to 0.7.1 but I cannot login any more. The error logs from `pm2` is as below:

```
ghost-0 xxxxxx - - [05/Oct/2016:17:25:38 +0000] "HEAD /ghost/ HTTP/1.0" - - "-" "okhttp/2.3.0"
ghost-0 xxxxxx - - [05/Oct/2016:17:25:40 +0000] "HEAD /ghost/ HTTP/1.0" 200 4238 "-" "okhttp/2.3.0"
ghost-0 xxxxxx - - [05/Oct/2016:17:26:02 +0000] "HEAD /ghost/ HTTP/1.0" 200 4238 "-" "okhttp/2.3.0"
ghost-0 ERROR: Please Sign In 
ghost-0  
ghost-0 xxxxxx - - [05/Oct/2016:17:26:02 +0000] "GET /ghost/api/v0.1/ghost/ HTTP/1.0" 401 73 "-" "okhttp/2.3.0"
ghost-0 xxxxxx - - [05/Oct/2016:17:26:02 +0000] "POST /ghost/api/v0.1/authentication/token HTTP/1.0" 401 - "-" "okhttp/2.3.0"
```

**Comment by vickychijwani** [comment]
Hi @baddwin! Sorry to hear about your problem. Let me ask the obvious questions first, since I've tested Quill with Ghost 0.7.x in the past (0.7.8 was the version I tested with, to be exact):

1. Are you able to login normally from a browser after upgrading?
2. Can you log out of Quill and re-enter your login details from scratch? (remove any pre-filled values first). It's possible that the upgrade caused some state in the app to go stale.
3. If you're getting an error on the login screen, can you tell me what it says, or upload a screenshot?

**Comment by vickychijwani** [comment]
@baddwin by the way, if you're ultimately going to upgrade to 0.11.1 from 0.7.1 as described [here](https://dev.ghost.org/ghost-0-11-1/), it might be better to finish that first and then try logging in through Quill again, because I test most thoroughly on the latest Ghost version.

**Comment by baddwin** [comment]
Okay, I've moved Ghost installation to Openshift instead, and it's version is now 0.7.5. But unfortunately, the error persists. To answer your questions:

1. Yes, I'm able to login normally from browser.
2. I don't know how to log out. But I've cleared app cache from Android app settings, also I've reinstalled Quill, but still the same. 
3. The error at Quill login is: *Unexpected error during signin. Please try again.*

**Comment by vickychijwani** [comment]
I can't find your blog's admin page. Isn't it supposed to be at <your site>/ghost ? When I visit that URL I get a 404 not found (both http and https).

Have you configured Ghost to make the admin page available at a different path, maybe? Because Quill can't handle that currently.

**Comment by baddwin** [comment]
I'm sorry, my blog  admin page is in `http://blogs-badwi.rhcloud.com/ghost/`

**Comment by vickychijwani** [comment]
Oh ok. So that seems fine. I can't see a way to debug this error without having access to Android logs. I'd like to propose a different way to solve this issue, if you agree: is it possible for you to "invite" me to your blog as an author (so I have no permissions except to create and edit my own posts) so I can try to login and debug this? Obviously it is up to you, if you think this is appropriate.

**Comment by baddwin** [comment]
Of course, invitation is fine. I have sent it to your email. Please also check your spam folder.

**Comment by vickychijwani** [comment]
@baddwin I found and fixed the bug! I've tested the fix on Ghost 0.7.5 (your blog), 0.6.3 (oldest supported version) and 0.11.1 (the latest version) just to be safe. I'm able to login to your blog now (you can remove me from it).

You should get an app update (v1.3.5) by tomorrow. Thanks for all your help in this! :smile:
